nautobot/extras/forms/forms.py

```diff
@@ -584,29 +584,26 @@ class Meta:
         model = ExternalIntegration
         fields = "__all__"
 
-        EXTRA_CONFIG_HELP_TEXT = """
+        HEADERS_HELP_TEXT = """
             Optional user-defined <a href="https://json.org/">JSON</a> data for this integration. Example:
             <pre><code>{
```

Comment:
`<pre><code>` seems redundant to me - does it render fine if we remove `<code>` tags?